src/adapters/github.js

```diff
@@ -147,11 +152,104 @@ class GitHub extends PjaxAdapter {
 
   // @override
   _getTree(path, opts, cb) {
-    this._get(`/git/trees/${path}`, opts, (err, res) => {
+    if (opts.repo.pullNumber) {
+      this._getPatch(opts, (err, res) => {
+        if (err) cb(err)
+        else cb(null, res)
+      })
+    }
+    else {
+      this._get(`/git/trees/${path}`, opts, (err, res) => {
+        if (err) cb(err)
+        else {
+          console.log(res.tree)
```

Comment:
Is this redundant?

Reply:
Yup, was debugging, need to remove this..